src/components/Menu/Menu.tsx

```diff
@@ -1,66 +1,34 @@
 import { ReactElement, useContext } from 'react';
-import { View } from '@aws-amplify/ui-react';
 import { MenuItem } from './MenuItem';
 import { Platform } from '@/data/platforms';
-import Link, { LinkProps } from 'next/link';
-import { LayoutContext } from '@/components/Layout';
 import { PageNode } from 'src/directory/directory';
 
 type MenuProps = {
   currentPlatform?: Platform;
   rootMenuNode: PageNode | undefined;
-  menuTitle: string;
-  menuHref: LinkProps['href'];
 };
 
 export function Menu({
   currentPlatform,
-  rootMenuNode,
-  menuTitle,
-  menuHref
+  rootMenuNode
 }: MenuProps): ReactElement {
-  const { menuOpen, toggleMenuOpen } = useContext(LayoutContext);
-
-  const onLinkClick = () => {
-    if (menuOpen) {
-      toggleMenuOpen(false);
-    }
-  };
-
   return (
     <nav className="menu">
       <ul className="menu__list">
-        <li className="menu__list-item">
-          <Link
-            onClick={onLinkClick}
-            className="menu__list-item__link"
-            href={menuHref}
-          >
-            <View
-              className="menu__list-item__link__inner"
-              marginBottom="16px"
-              fontWeight={700}
-            >
-              {menuTitle}
-            </View>
-          </Link>
-        </li>
-        <ul className="menu__list">
```

Comment:
Removed this `ul` element because it was previously nesting a `ul` within a `ul`. The `MenuItem` component already returns `li` elements and so everything still gets nested correctly